tests/test_chrono.py

```diff
@@ -9,21 +9,23 @@
 def test_chrono_system_clock():
 
     # Get the time from both c++ and datetime
+    date0 = datetime.datetime.today()
     date1 = m.test_chrono1()
     date2 = datetime.datetime.today()
 
     # The returned value should be a datetime
     assert isinstance(date1, datetime.datetime)
 
     # The numbers should vary by a very small amount (time it took to execute)
+    diff_python = abs(date2 - date0)
     diff = abs(date1 - date2)
 
     # There should never be a days difference
     assert diff.days == 0
 
     # Since datetime.datetime.today() calls time.time(), and on some platforms
-    # that has 1 second accuracy, we should always be less than 2 seconds.
-    assert diff.seconds < 2
+    # that has 1 second accuracy, we compare this way
+    assert int(diff.seconds) <= int(diff_python.seconds)
```

Comment:
Why `int` here, though?

Reply:
I didn't check to make sure it was forced to be an int already. Is it?

Reply:
I think it is. The docs say it is `in range(60)`.

Reply:
OK. The idea was mainly that even if it was something else, `<=` still ought to work? :-) But good!